### PR TITLE
Adding closing-slide component.

### PR DESCRIPTION
**Pull request by aorinevo**
* Add closing-slide component.
* Uses multi content projection.
* Currently only implemented on Typescript and Bootstrap sections.

### CODE REVIEW

**Comment by kirjs** [comment]
Thanks! Add a screenshot?

**Comment by aorinevo** [comment]
screen shot 2017-05-02 at 9 17 47 pm
screen shot 2017-05-02 at 9 18 09 pm

File: src/app/presentation/closing-slide/closing-slide.component.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+.last-slide {
+  display: flex;
+  width: 1200px;
+  height: 640px;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
```
Comment (kirjs) [inline src/app/presentation/closing-slide/closing-slide.component.css]: Don't most of those props basically come for free if it's in the slide?

**Comment by aorinevo** [comment]
See Issue #94

File: src/app/presentation/closing-slide/closing-slide.component.css (hunk 1 above)
Comment (aorinevo) [inline src/app/presentation/closing-slide/closing-slide.component.css]: I needed this because I was using flex.

**Pull request merged**